ks-nav/CONTRIBUTING.md

```diff
@@ -0,0 +1,19 @@
+# Contributing to nav
+
+:+1::tada: First off, thanks for taking the time to 
+contribute! :tada::+1:
+
+The following is a set of guidelines for contributing to nav.
+The nav utility is hosted in the 
+[nav repository](https://github.com/alessandrocarminati/nav) 
```

Comment:
Should this refer to the ELISA GitHub repo instead?

Reply:
Currently, I am developing the tool, which is not yet finished in the repository that is listed in the README file. The link's intent was to provide the source where a user could find the newest version. If required, I can change them instantly.

Reply:
I think we should move everything to the ELISA github and remove personal repos